harness/determined/common/experimental/determined.py

```diff
@@ -65,6 +65,15 @@ def create_experiment(
         config: Union[str, pathlib.Path, Dict],
         model_dir: str,
     ) -> experiment.ExperimentReference:
+        """
+        Get the :class:`~determined.experimental.ExperimentReference` representing the
+        experiment created with the provided config and model direcotry.
```

Comment:
The way this is phrased makes it sound like `create_experiment()` is not actually creating anything.  Rephrase to state what it does and what it returns.